On The Map/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,348 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="10116" systemVersion="15E65" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" initialViewController="lQm-Eo-yEg">
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="10085"/>
+        <capability name="Constraints to layout margins" minToolsVersion="6.0"/>
+        <capability name="Navigation items with more than one left or right bar item" minToolsVersion="7.0"/>
+    </dependencies>
+    <scenes>
+        <!--Login-->
+        <scene sceneID="uLx-ei-Dhb">
+            <objects>
+                <viewController id="lQm-Eo-yEg" userLabel="Login" customClass="OTMLoginViewController" customModule="On_The_Map" customModuleProvider="target" sceneMemberID="viewController">
```

Comment:
As you can see, the storyboard files are impossible to read. Android's XML layout files are a lot easier to read / write / diff.